docs/webhooks/events.md

```diff
@@ -254,21 +254,21 @@ Type | Description
 
 Property | Value | Description
 ---------|-------|------------
-`answers` | Hash | Survey question and responses
 `submitted_at` | date-time | Date/time response was submitted (EST/EDT or GMT -5/GMT -4)
-`response_id` | String | Survey external Response ID
-`url` | Hash | URL variable value for Survey
 `main_indicator` | Integer | Main question result
 `main_indicator_scaled` | Float | Main question result scaled to 0..1
 `intervention_id` | String | ID of the intervention related to this survey
 `survey_id` | String | ID of the survey
 `source_id` | String | ID of the source related to this survey
 `user_id` | String | ID of the user related to this survey
-`context_data` | Hash | Context_data is present only if the content has context_data associated. The context_data hash keys are the custom fields keys.
+`questions` | Array of Hashes | List of questions containing the displayed `text` of the question, its `foreign_id` (from Alchemer) and their `replies` in the form of an Array. Each one of the `replies` entry contains two keys: `text` and `value` of the customer reply.
```

Comment:
this webhook is not only about Alchemer so we should be more generic:
```suggestion
`questions` | Array of Hashes | List of questions containing the displayed `text` of the question, its `foreign_id` (from the third party survey supplier) and their `replies` in the form of an Array. Each one of the `replies` entry contains two keys: `text` and `value` of the customer reply.
```

Reply:
🤦  thanks